src/components/departure-list/section-items/line.tsx

```diff
@@ -44,7 +44,7 @@ import {
   View,
 } from 'react-native';
 import {TouchableOpacity} from 'react-native-gesture-handler';
-import {NearbyScreenNavigationProp} from '../Nearby';
+import {NearbyScreenNavigationProp} from '../../../screens/Nearby/Nearby';
```

Comment:
This can be `@atb/screens/Nearby/Nearby` maybe should it be exported from index so it could be `@atb/screens/Nearby`